model-optimizer/mo/utils/shape.py

```diff
@@ -38,13 +38,13 @@ def get_canonical_axis_index_node(rank: Node, axis: int) -> Node:
     graph = rank.graph
     name = rank.soft_get('name', rank.id)
     if axis < 0:
-        axis = Const(graph, {'name': name + '/negative_axis', 'value': int64_array([axis])}).create_node()
```

Comment:
The function `get_canonical_axis_index_node` is used in the `AtenToEmbeddingBag` and `MXRepeatReplacer` transformations. The fact that you changed it here may result in the different behaviour of these transformation.

Reply:
Yes, I know but it is wrong for them. As I understand e2e tests with them pass. Now we have a new issue with `Unsquueze`.
To implement a new transformation for `Range` with 1d input tensors seems wrong to me since that `Range` is created by MO and this is not MXNet specifics